--- imb/3.2.4/src/make_mpich.orig	2014-05-02 12:44:36.633388115 -0400
+++ imb/3.2.4/src/make_mpich	2014-05-02 12:45:21.442295775 -0400
@@ -1,17 +1,10 @@
 # Enter root directory of mpich install
-MPI_HOME=
 
-MPICC=$(shell find ${MPI_HOME} -name mpicc -print)
 
 NULL_STRING :=
-ifneq (,$(findstring /bin/mpicc,${MPICC}))
-MPI_INCLUDE := -I${MPI_HOME}/include
-else
-$(error Variable MPI_HOME="${MPI_HOME}" does not seem to contain a valid mpicc)
-endif
 LIB_PATH    =
 LIBS        = 
-CC          = ${MPI_HOME}/bin/mpicc 
+CC          = mpicc 
 OPTFLAGS    = -O3
 CLINKER     = ${CC}
 LDFLAGS     =
--- imb/3.2.4/src/Makefile.base.make	2014-05-02 13:08:24.180791986 -0400
+++ imb/3.2.4/src/Makefile.base	2014-05-02 13:08:56.311127164 -0400
@@ -64,7 +64,7 @@ clean:
 	/bin/rm -f *.o *~ PI* core IMB-IO IMB-EXT IMB-MPI1 IMB-IO.mic IMB-EXT.mic IMB-MPI1.mic exe_io exe_ext exe_mpi1
 
 .c.o:
-	$(CC) $(MPI_INCLUDE) $(CPPFLAGS) -D$(CPP) $(OPTFLAGS) -c $*.c
+	$(CC) $(CPPFLAGS) -D$(CPP) $(OPTFLAGS) -c $*.c
 
 # DEPENDENCIES
 IMB_declare.h:	IMB_settings.h IMB_comm_info.h IMB_settings_io.h IMB_bnames_mpi1.h \
