OMB: disable collective async

To build mpitests against old MPI implementations that don't provide
MPI_Ibarrier etc.

diff -up osu-micro-benchmarks-5.4/mpi/collective/Makefile.in.orig osu-micro-benchmarks-5.4/mpi/collective/Makefile.in
--- osu-micro-benchmarks-5.4/mpi/collective/Makefile.in.orig	2017-11-03 23:06:44.176756058 +0100
+++ osu-micro-benchmarks-5.4/mpi/collective/Makefile.in	2017-11-03 23:07:05.677306833 +0100
@@ -93,13 +93,7 @@ collective_PROGRAMS = osu_alltoallv$(EXE
 	osu_reduce_scatter$(EXEEXT) osu_barrier$(EXEEXT) \
 	osu_reduce$(EXEEXT) osu_allreduce$(EXEEXT) \
 	osu_alltoall$(EXEEXT) osu_bcast$(EXEEXT) osu_gather$(EXEEXT) \
-	osu_allgather$(EXEEXT) osu_scatter$(EXEEXT) \
-	osu_iallgather$(EXEEXT) osu_ibcast$(EXEEXT) \
-	osu_ialltoall$(EXEEXT) osu_ibarrier$(EXEEXT) \
-	osu_igather$(EXEEXT) osu_iscatter$(EXEEXT) \
-	osu_iscatterv$(EXEEXT) osu_igatherv$(EXEEXT) \
-	osu_iallgatherv$(EXEEXT) osu_ialltoallv$(EXEEXT) \
-	osu_ialltoallw$(EXEEXT)
+	osu_allgather$(EXEEXT) osu_scatter$(EXEEXT)
 @CUDA_KERNELS_TRUE@am__append_1 = ../../util/kernel.cu
 @BUILD_PROFILING_LIB_TRUE@@EMBEDDED_BUILD_TRUE@am__append_2 = $(top_builddir)/../lib/lib@PMPILIBNAME@.la
 @OPENACC_TRUE@am__append_3 = -acc
